### PR TITLE
Created recs for Fremont, CA

### LINKED ISSUE #3: Create a new file for githubteacher's hometown (San Diego, CA)
## San Diego, CA
- [x] Create a branch
- [x] Add commits (make changes)
- [x] Open a Pull Request
- [x]  Collaboration
- [x] Seek approval
- [x] Merge PR

### PR DESCRIPTION
**Pull request by shahm21**
Resolve #3

### CODE REVIEW

File: place/monarch-fremont.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+### Places to Eat - 
+- Kabila
+- Mosimo
+- Sakoon
+
+### Places to Avoid 
+- None
+
+### Places to GO 
+- Lake Elizabeth
```
Comment (azzurri2015) [inline place/monarch-fremont.md:12]: Need to check this out!

File: place/monarch-fremont.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
+# Fremont, CA
+
+### Places to Eat - 
+- Kabila
+- Mosimo
+- Sakoon
```
Comment (azzurri2015) [inline place/monarch-fremont.md]: What type of food is this place?

**Review by azzurri2015 — APPROVED** [review]

File: place/monarch-fremont.md (hunk 1 above)
Comment (shahm21) [reply place/monarch-fremont.md:12]: :+1:

**Review by azzurri2015 — APPROVED** [review]

**Comment by azzurri2015** [comment]
Nice job, ready to go

**Pull request merged**